core/new-gui/src/app/workspace/service/validation/validation-workflow.service.ts

```diff
@@ -178,7 +178,7 @@ export class ValidationWorkflowService {
     const actualOutputNum = texeraGraph.getOutputLinksByOperatorId(operatorID).length;
 
     const satisfyInput = requiredInputNum === actualInputNum;
-    const satisyOutput = requiredOutputNum === actualOutputNum;
+    const satisyOutput = true; //requiredOutputNum === actualOutputNum;
```

Comment:
remove the commented out lines. this applies for other places as well, remove the commented out code if it's no longer used.